tests/unit/amo/pages/TestAddon.js

```diff
@@ -1096,10 +1098,17 @@ describe(__filename, () => {
 
   it('hides banner on non firefox clients and displays firefox download button', () => {
     const root = shallowRender({
-      getClientCompatibility: getClientCompatibilityFalse,
+      getClientCompatibility: getFakeClientCompatibility(false),
     });
     expect(root.find(AddonCompatibilityError)).toHaveLength(0);
-    expect(root.find(Button)).toHaveLength(1);
+    expect(root.find(GetFirefoxButton)).toHaveLength(1);
+  });
+
+  it('hides GetFirefoxButton for Firefox', () => {
+    const root = shallowRender({
+      getClientCompatibility: getFakeClientCompatibility(true),
+    });
+    expect(root.find(GetFirefoxButton)).toHaveLength(0);
```

Comment:
assuming you do make the change above, maybe this is not needed anymore?

Reply:
This change is what I am referring to in the above comment: https://github.com/mozilla/addons-frontend/pull/6925#discussion_r233978406